### DISCUSSION TITLE
Associate AppImageLauncher with `.AppImage` files

### DISCUSSION THREAD

**Discussion by ernstki** [TheAssassin/AppImageLauncher#635] [Ideas]
I did search the issue tracker, no mention of this.

It seems theoretically possible to have (the non-"lite" version of) AppImageLauncher take over the association of `.AppImage` files and just "do the right thing" when they're double-clicked. That is, prompt the user, then set the `.AppImage` file executable, then do the normal integration steps that AppImageLauncher is known for.

Curious why this isn't already the case. Was this option discussed/explored already?

I wasn't able to demo the non-lite version of AppImage without building from source (which I don't have time to do at the moment). So if AppImageLauncher **already** does this, my bad!

**Comment by TheAssassin** [comment]
The AppImageLauncher packages should already set  up MIME integration just the way you'd like it to be.

Please run `xdg-mime query default application/vnd.appimage` and check if it reports `appimagelauncher.desktop`.

**Reply by ernstki** [reply]
Thanks, I'll keep that in mind if I go back to using `.AppImage` versions of any apps in the future.

**Comment by TheAssassin** [comment]
Closing due to the lack of a response.
